docker-compose.yaml

```diff
@@ -499,7 +499,7 @@ services:
     ports:
       - '5601:5601'
     environment:
-      KEYCLOAK_URL: http://docker.for.mac.localhost:8088
+      KEYCLOAK_URL: http://${KEYCLOAK_URL:-docker.for.mac.localhost:8088}
```

Comment:
can you explain that change?

Reply:
it comes from #1526 .
GNU/Linux doesn't know `docker.for.mac.localhost` and both `logs-db` and `logs-db-ui` dont' start properly.
To test lagoon on my local, I had to "merge" the changes @smlx made in the PR mentioned.

Reply:
okay, but let's remove it from this PR please, we want to keep them separate

Reply:
Removed